src/components/StrollButton.tsx

```diff
@@ -46,7 +46,7 @@ const StrollButton: React.FC<StrollButtonProps> = ({
 				}}
 				accessibilityLabel={isShowingSelections ? "Hide selections" : "Show selections"}
 				accessibilityHint={`Current selections: ${buildSelectionsText(selections)}`}
-				iconName={isShowingSelections ? "chevron-down" : "chevron-up"}
+				iconName={isShowingSelections ? "chevron-down" : "filter"}
```

Comment:
because the filter icon is not the same width as the chevron, the button moves a little bit when you press it now. is that okay?